### PR TITLE
Sh person mnc008

### PR DESCRIPTION
**Pull request by SusanneHummel**
Dear both,

Here are five persons related to MNC-008.

Best,
Sanne

### CODE REVIEW

File: new/PRS13274AmIya.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13274AmIya" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾAmda ʾIyasus</title>
```
Comment (MassimoVilla) [inline new/PRS13274AmIya.xml:8]: Etymologically it is ʿAmda. Do we standardize proper names although their spellings in the ms are non-etymological?

File: new/PRS13281HayMar.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13281HayMar" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Ḫāyla Māryām</title>
```
Comment (MassimoVilla) [inline new/PRS13281HayMar.xml:8]: Also here, the etymological spelling is Ḫayla. I do not know the policy of BM in these cases.

**Review by MassimoVilla — CHANGES_REQUESTED** [review]

File: new/PRS13274AmIya.xml (hunk 1 above)
Comment (antobrita) [reply new/PRS13274AmIya.xml:8]: > Etymologically it is ʿAmda. Do we standardize proper names although their spellings in the ms are non-etymological?

I would suggest to standardize the proper names.

File: new/PRS13274AmIya.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-12-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አምደ፡ ኢየሱስ፡</persName>
```
Comment (antobrita) [inline new/PRS13274AmIya.xml:48]: Here too and in the transliteration

File: new/PRS13274AmIya.xml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAmda ʾIyasus</persName>
+                    <floruit notBefore="1800" notAfter="1900"/>
+                    <floruit/>
+                    <occupation type="literary">scribe</occupation>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13274AmIya" passive="MNC008"></relation>
+                    <relation name="saws:isCopierOf" active="PRS13274AmIya" passive="MNC008#ms_i1">
+                        <desc>The name of ʾAmda ʾIyasus is mentioned as a scribe in the
```
Comment (antobrita) [inline new/PRS13274AmIya.xml:57]: Also here

**Review by antobrita — APPROVED** [review]

File: new/PRS13274AmIya.xml (hunk 1 above)
Comment (SusanneHummel) [reply new/PRS13274AmIya.xml:8]: @thea-m or @DariaElagina Waht is the BM policy for Person records: name shall be standardized according to Dillmann ?

Comment (thea-m) [reply new/PRS13274AmIya.xml:8]: There is no policy. If you decide to standardise, I think it would be good to state it.

Comment (SusanneHummel) [reply new/PRS13274AmIya.xml:8]: To state it in the record?

Comment (thea-m) [reply new/PRS13274AmIya.xml:8]: I think that would make sense

Comment (SusanneHummel) [reply new/PRS13274AmIya.xml:8]: I will change the person records and add < note. However, in the manuscrupt record, I follow the wrting of the manuscript, correct?

**Review by MassimoVilla — APPROVED** [review]

**Pull request merged**